### DISCUSSION TITLE
VcDrawings裡的mouseEvt,在繪制polygon後,無_vcPolylineIndx

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#345] [Q&A]
VcDrawings裡的mouseEvt,繪制圖形後,polygon與polyline的_vcPolylineIndx是undefined
其他的圖形,點或圖標皆有回傳

![2023-02-10_133016](https://user-images.githubusercontent.com/5283322/218009361-c4909f09-2bde-42a1-b27f-86e01c5b9d8c.png)

**Comment by zouyaoji** [comment]
polygon比较特殊，用的自定义图元 https://github.com/zouyaoji/vue-cesium/blob/dev/packages/shared/src/PolygonPrimitive.ts，如果你自己需要获取_vcPolylineIndx ，得通过 cesiumObject._vcParent._vcPolylineIndex 拿到。
